spec/checker_jobs/notifiers/bugsnag_spec.rb

```diff
@@ -15,42 +15,38 @@
   end
 
   it "notifies bugsnag" do
-    allow(::Bugsnag::Notification).to receive(:deliver_exception_payload)
+    allow(::Bugsnag).to receive(:deliver_notification)
     perform
-    expect(::Bugsnag::Notification).to have_received(:deliver_exception_payload).once
+    expect(::Bugsnag).to have_received(:deliver_notification).once
   end
 
   describe "notify's resulting payload" do
-    subject(:notify_payload) do
-      payload = nil
+    subject(:notify_report) do
+      report = nil
 
-      # NOTE: To get compatibility with Bugsnag 6, we'll get the payload from elsewhere:
-      # Bugsnag::Delivery[configuration.delivery_method].deliver(_, payload, _, _)
-      allow(::Bugsnag::Notification).
-        to receive(:deliver_exception_payload).
-        and_wrap_original do |_method, _, p, _, _|
-          payload = p[:events].first
-        end
+      allow(::Bugsnag).
+        to receive(:deliver_notification).
+        and_wrap_original { |_, r| report = r }
 
       perform
 
-      payload
+      report
     end
 
     # rubocop:disable RSpec/ExampleLength
     it "is an Error and have an explicit message", aggregate_failures: true do
       # We hook deep into Bugsnag to ensure we send the right payload.
       # This ensure we're only compatible with Bugsnag 5 and not Bugsnag 6.
```

Comment:
Did the changes bellow invalid what is said in this comment ?
In either case shouldn't it be reflected in the `gemspec` version requirements ?

Reply:
👋

Reply:
Sorry yes I did miss it. Well, nothing in the runtime code changed between these major versions, it's just the specs. So technically you could work with Sidekiq 5 in development just fine, though the specs would fail.

Since it's only development deps, I decided to do as you suggest and adapt the `gemspec`.